### ISSUE TITLE
[Supporting the platform – Make Code – Create a project ] Focus indicator is not visible on 'Create & Close' button controls after applying high contrast (black/white) theme.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3562]
**User Experience:**
If the desired component, on which keyboard operations will interact at any point in time is not visually determined, people with attention limitations, short term memory limitations, or limitations in executive processes will find it difficult to discover where the focus is located in high contrast theme.

**Note**: User credentials should NOT be included in the bug.

**Pre-requisites:**
Enable high contrast mode. Settings->Ease of Access->Color & High contrast

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing under "My Project" section in Home page and select 'New Project' Button.
4. Navigate all controls under 'Create a Project' pop-up.
5. Verify the focus visibility on 'Create & Close' button controls after applying high contrast black/white theme.

**Actual Result:**
Keyboard focus indicator is not visible on 'Create & Close' button controls after applying high contrast (black/white) theme under 'Create a project' pop-up and due to this, visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus indicator should be visible on 'Create & Close' button controls after applying high contrast (black/white) theme under 'Create a project' pop-up, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev1, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.3.1, #Win10-Edge(Chromium)

[WCAG4.3.1_Create a project_HC.zip](https://github.com/microsoft/pxt-microbit/files/5479797/WCAG4.3.1_Create.a.project_HC.zip)
